Source/WebInspectorUI/UserInterface/Models/FontStyles.js

```diff
@@ -138,7 +138,10 @@ WI.FontStyles = class FontStyles
 
         const createIfMissing = true;
         let cssProperty = this._effectiveWritablePropertyForName(targetPropertyName, createIfMissing);
+
         cssProperty.rawValue = targetPropertyValue;
+        if (this._nodeStyles._propertyNameToEffectivePropertyMap[cssProperty.name]?.important)
+            cssProperty.priority = "important";
```

Comment:
Setting `WI.CSSProperty.priority = important` won't actually update the `rawValue` and or it to the backend (i.e. the inspected page) to reflect the style change. 

You were on the right track with concatenating `targetPropertyValue += " !important"`, but that needs to be done before assigning `WI.CSSProperty.rawValue`. 

That setter is the one that triggers the operations to update the style text and send it to the backend. 
Follow the operations called from here:

https://github.com/WebKit/WebKit/blob/54140a7b23efcb75955d08e7e7882371d1af3d99/Source/WebInspectorUI/UserInterface/Models/CSSProperty.js#L383-L397